troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -21,29 +26,42 @@ export default React.createClass({
         showAvailability: React.PropTypes.bool
     },
 
-    getDefaultProps: function() {
+    getDefaultProps() {
         return {
             showAvailability: true,
             editable: true
         }
     },
 
-    onEditClicked: function() {
+    onCardClick() {
+        let isOpen = this.state.isOpen
+            ? false : true;
```

Comment:
This is toggling the state `isOpen` between true and false. I could comment that if it isn't clear. Or if you have a better way? I don't need to coerce `this.state.isOpen` because it is defined as false. 

I'm confused by the mention of `!!`? 
Are you just more generally speaking to ways to coerce a bool?